config/permissions.cfg

```diff
@@ -29,9 +30,12 @@
 # Default behaviour:
 # ------------------
 #   - create missing resources if supported by the service (and tree automatically resolvable), then apply permissions.
-#   - create missing user/group if required (default user created: (group: anonymous, password: 12345).
-#   - applicable service, user or group is missing, corresponding permissions are ignored and not updated.
-#   - unknown actions are ignored and corresponding permission are not updated, unspecified action resolves to 'create'.
+#   - create missing user/group if required (default user created: (group: anonymous, password: 12345) ).
+#   - if applicable service, user or group is missing, corresponding permissions are ignored and not updated.
+#   - if resolved resource during 'create' action results into an ambiguity (eg: cannot infer which type to apply),
```

Comment:
"if resolved resource during 'create' action results into an ambiguity (eg: cannot infer which type to apply)" So that was the root cause of the failure in PAVICS?

Reply:
the parsing of `read-deny-recursive` was compared against the plain permssion `read` without considering the access/scope modifiers